<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CondaNewEnvironmentDataDialog</class>
 <widget class="QDialog" name="CondaNewEnvironmentDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Conda Environment</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Logical Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="E5ClearableLineEdit" name="nameEdit">
     <property name="toolTip">
      <string>Enter a unique name for the virtual environment to register it with the Virtual Environment Manager</string>
     </property>
     <property name="placeholderText">
      <string>Name for registration of the virtual environment</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Conda Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="E5ClearableLineEdit" name="condaNameEdit">
     <property name="toolTip">
      <string>Enter the name of the virtual environment in Conda</string>
     </property>
     <property name="placeholderText">
      <string>Name of the virtual environment in Conda</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="requirementsLabel">
     <property name="text">
      <string>Requirements File:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="E5PathPicker" name="requirementsFilePicker" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5ClearableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>E5Gui/E5LineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CondaNewEnvironmentDataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CondaNewEnvironmentDataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
